src/components/Patient/ShiftCreate.tsx

```diff
@@ -261,142 +263,138 @@ export const ShiftCreate = (props: patientShiftProps) => {
   });
 
   return (
-    <Page
-      title={"Create Shift Request"}
-      crumbsReplacements={{
-        [facilityId]: { name: patientData?.facility_object?.name || "" },
-        [patientId]: { name: patientData?.name || "" },
-      }}
-      backUrl={`/facility/${facilityId}/patient/${patientId}`}
+    <SlideOver
+      open={props.open}
+      setOpen={props.setOpen}
+      title="Create Shift Request"
+      slideFrom="right"
+      dialogClass="md:w-[780px]"
     >
-      <Card className="mx-auto mt-4 flex w-full max-w-3xl flex-col px-8 py-5 md:px-16 md:py-11">
-        <TextFormField
-          {...field("refering_facility_contact_name")}
-          label="Name of Contact person at the current facility"
-          required
-        />
-
-        <PhoneNumberFormField
-          {...field("refering_facility_contact_number")}
-          label="Contact person phone number"
-          required
-          types={["mobile", "landline"]}
-        />
-
-        {careConfig.wartimeShifting && (
-          <div>
-            <FieldLabel required>
-              Name of shifting approving facility
-            </FieldLabel>
+      <Card className="mx-auto mt-4 flex flex-col ">
+        <div className="lg:grid gap-4 lg:grid-cols-2 ">
+          <TextFormField
```

Comment:
Should switch over to using react hook forms and shadcn; they are being added in `form-field-v1` branch; check it out for examples.

Reply:
@Jacobjeevan You mean like should i replace the form fileds files in my branch with `form-field-v1` ?

Reply:
I have looked into `src/components/Form/FormFields` almost all used files are same

Reply:
> @Jacobjeevan You mean like should i replace the `src/components/Form/FormFields` files in my branch with `form-field-v1` ?

No I meant switch to using React Hook Forms and Shadcn's form in ShiftCreate.

Reply:
Hey @Jacobjeevan , i have done changes related to `useQuery `and `useMutation `tanstack. 

Regarding shadcn's form, the shadcn form and components are present in `form-field-v1`  branch but not present in `develop` branch. should i create files and place the code in `develop `branch from `form-field-v1` branch ?

Is there any example usage i can refer to?

Reply:
> Hey @Jacobjeevan , i have done changes related to `useQuery `and `useMutation `tanstack.
> 
> Regarding shadcn's form, the shadcn form and components are present in `form-field-v1` branch but not present in `develop` branch. should i create files and place the code in `develop `branch from `form-field-v1` branch ?
> 
> Is there any example usage i can refer to?

Yep, you can refer to `ScheduleExceptionForm` and `ScheduleTemplateForm` for examples. I'll be pushing my own changes to `PatientRegistration` (src/pages/appointments/PatientRegistration tonight, so you can check that as well.

You can add Shadcn's components for each (Input, Date Picker etc).